src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"red", "green", "blue", "yellow", "black"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
The `Random` object can be a class-level variable instead of being instantiated every time `getRandomColor()` is called. This would be more efficient as it will reduce the overhead of object creation on each method call.